src/core/stmt.c

```diff
@@ -1537,10 +1537,10 @@ static SQLRETURN _stmt_set_row_bind_type(stmt_t *stmt, SQLULEN row_bind_type)
 
 static SQLRETURN _stmt_set_param_bind_type(stmt_t *stmt, SQLULEN param_bind_type)
 {
-  // if (param_bind_type != SQL_BIND_BY_COLUMN) {
-  //   stmt_append_err(stmt, "HY000", 0, "General error:only `SQL_BIND_BY_COLUMN` is supported now");
-  //   return SQL_ERROR;
-  // }
+  if (param_bind_type != SQL_BIND_BY_COLUMN) {
```

Comment:
SQL_BIND_BY_COLUMN => SQL_PARAM_BIND_BY_COLUMN where param-bind is related

and search/replace everywhere else